config/environment.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(environment) {
     },
 
     fastboot: {
-      hostWhitelist: ['emberguides.stonecircle.io', 'localhost:4200']
+      hostWhitelist: ["localhost:4200", "localhost:7784"]
```

Comment:
can you herokuapp & fastly.net to hostWhiteList similar to other apps?

Reply:
this isn't necessary because of the fact that we're only running fastboot at build time. We should remove herokuapp and fastly.net from deprecations app if it's there 👍